lib/passport-ldapauth/strategy.js

```diff
@@ -123,6 +123,8 @@ var verify = function() {
  *                         (default: 'Invalid username/password')
  * - `userNotFound`  flash message when LDAP returns no error but also no user
  *                   (default: 'Invalid username/password')
+ * - `ConstraintViolationError`  flash message when user account is locked
```

Comment:
This is spelled with lower case `C` in the code (`constraintViolation`), please update documentation here and in README.md.